prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -98,7 +146,7 @@ class LivdTableDownload : CliktCommand(
                 if (c?.lowercase() == "n") {
```

Comment:
Remove this check for overwriting the downloaded file and just write it to a temp file.  I get asked twice if I want to overwrite the file.  You may want to look at the output of the tool to make sure the messages flow well.  For example, there are a few SUCCESS messages that may be confused that the whole process was a success.

Reply:
Done.